Makefile

```diff
@@ -244,7 +244,7 @@ $(DEP_BIN):
 ifeq ($(UNAME_S),Darwin)
```

Comment:
I can't see where `$(UNAME_S)` is set if we are calling `make build` goal. I believe we should set it so that user doesn't have to explicitly set using `export UNAME_S=Darwin`. So the simple thing to do is define `UNAME_S=$(shell uname -s)`

Reply:
@dipak-pawar Added UNAME_S variable initialization.